generative_models/3D-GAN_PyTorch/README.md

```diff
@@ -0,0 +1,117 @@
+# **PyTorch implementation of 3D-GAN**
+## Usage
+```bash
+> python main.py --arguments
```

Comment:
you may add output of `python main.py --help` here, to indicate available arguments

Reply:
Yes I will do that